content/ja/security/cloud_workload_security/backend.md

```diff
@@ -3199,4 +3199,4 @@ CWS のログは、以下の JSON スキーマを持ちます。
 
 
 [1]: /ja/security/cloud_workload_security/
-[2]: /ja/security/cloud_workload_security/agent_expressions
\ No newline at end of file
+[2]: /ja/security/cloud_workload_security/agent_expressions
```

Comment:
weird diff 😕

Reply:
yeah the "rich diff" doesn't recognize this line as changed; not sure if this is a github regression or if it's tracking something real.

Reply:
I bet you added a Mongolian vowel separator just to mess with me

Reply:
That would have shown up on the rich diff. 💚